package.json

```diff
@@ -2,6 +2,10 @@
   "name": "insights-dashboard",
   "version": "1.0.0",
   "private": false,
+  "engines": {
+    "node": ">=16.0.0",
```

Comment:
If not already you should also update the `travis.yml` to run on 16 so your production build uses the same version.

Reply:
Good point @Hyperkid123 💯 have pushed the change in the latest commit.